### PR TITLE
Add Chillhop connector

### PR DESCRIPTION
**Pull request by folsomwg**
**Describe the changes you made**
Add Chillhop connector, requested here: https://github.com/web-scrobbler/web-scrobbler/issues/2232

### CODE REVIEW

File: src/connectors/chillhop.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+Connector.playerSelector = '#player-controlls';
```
Comment (folsomwg) [inline src/connectors/chillhop.js]: This isn't a typo, it's how it is in the page.

File: src/connectors/chillhop.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Connector.pauseButtonSelector = '.fa-pause';
+Connector.durationSelector = '.track-length';
+Connector.artistSelector = '.jp-artists';
+
+const trackSelector = '.jp-title';
+Connector.getTrack = () => {
+	return Util.getTextFromSelectors(trackSelector);
```
Comment (alexesprit) [inline src/connectors/chillhop.js]: Why don't you use `trackSelector` directly?

File: src/connectors/chillhop.js (hunk 1 above)
Comment (alexesprit) [reply src/connectors/chillhop.js]: Could you add a comment about this in the code?

File: src/connectors/chillhop.js (hunk 2 above)
Comment (folsomwg) [reply src/connectors/chillhop.js]: Fixed.

File: src/connectors/chillhop.js (hunk 1 above)
Comment (folsomwg) [reply src/connectors/chillhop.js]: Done.

**Pull request merged**